.prettierrc

```diff
@@ -4,5 +4,6 @@
   "trailingComma": "all",
   "proseWrap": "always",
   "arrowParens": "avoid",
-  "endOfLine": "auto"
+  "endOfLine": "auto",
+  "printWidth": 160
```

Comment:
I ask you to revert this to 80. Wide lines make it very hard to work in multi-pane environment.